examples/flutter_gallery/lib/demo/typography_demo.dart

```diff
@@ -49,15 +49,15 @@ class TypographyDemo extends StatelessWidget {
     final TextTheme textTheme = Theme.of(context).textTheme;
     final List<Widget> styleItems = <Widget>[
       if (MediaQuery.of(context).size.width > 500.0)
-        TextStyleItem(name: 'Display 4', style: textTheme.display4, text: 'Light 112sp'),
-      TextStyleItem(name: 'Display 3', style: textTheme.display3, text: 'Regular 56sp'),
-      TextStyleItem(name: 'Display 2', style: textTheme.display2, text: 'Regular 45sp'),
-      TextStyleItem(name: 'Display 1', style: textTheme.display1, text: 'Regular 34sp'),
-      TextStyleItem(name: 'Headline', style: textTheme.headline, text: 'Regular 24sp'),
-      TextStyleItem(name: 'Title', style: textTheme.title, text: 'Medium 20sp'),
-      TextStyleItem(name: 'Subheading', style: textTheme.subhead, text: 'Regular 16sp'),
-      TextStyleItem(name: 'Body 2', style: textTheme.body2, text: 'Medium 14sp'),
-      TextStyleItem(name: 'Body 1', style: textTheme.body1, text: 'Regular 14sp'),
+        TextStyleItem(name: 'Headline 1', style: textTheme.headline1, text: 'Light 112sp'),
```

Comment:
Should the typography demo use the 2018 values for font weight and size instead now?

Reply:
That's a good idea although an update to the typography demo would be probably be best as separate PR